web-client/src/modules/create/components/PostMap.tsx

```diff
@@ -0,0 +1,134 @@
+import { Button, Select } from 'antd';
+import React, { useState } from 'react';
+import TitleWithOrangeUnderline from 'src/components/TitleWithOrangeUnderline/TitleWithOrangeUnderline';
+import { IUserAddress } from 'src/models/users/privilegedInformation';
+import { NewAddressModal } from 'src/modules/create/components/NewAddressModal';
+import { COLORS } from 'src/theme/colors';
+import styled from 'styled-components';
+
+import WebClientMap from '../../../components/WebClientMap/WebClientMap';
+
+const PostMap: React.FC<PostMapProps> = ({
+  addresses,
+  postLocation,
+  postDetails,
+  setPostLocation,
+  nextHandler,
+  prevHandler,
+}) => {
+  const [showAddressModal, setShowAddressModal] = useState<boolean>(false);
+  const modalSuccess = value => {
+    setPostLocation(value);
+    setShowAddressModal(false);
+  };
+  const closeModal = () => {
+    setShowAddressModal(false);
+  };
+
+  const handleNameChange = value => {
```

Comment:
nit: consider renaming to handleAddressChange as it is more specific

Reply:
Done